### ISSUE TITLE
Implement corpus operators supported by ParallelTextCorpus for NParallelTextCorpus

### ISSUE THREAD

**Issue by Enkidu93** [sillsdev/machine#272]

**Comment by Enkidu93** [comment]
Where was this completed, @johnml1135 ? This involves adding new methods to the CorpusExtensions in parallel with those that exist for ParallelTextCorpus and was not part of my NParallelTextCorpus PR.

**Comment by johnml1135** [comment]
My fault - I thought that it was already done.  Is this needed for anything?  We are currently doing operations on arrays of corpora rather than using NParrallelTextCorpus directly.  Therefore, are the corpus extensions needed?

**Comment by Enkidu93** [comment]
> My fault - I thought that it was already done. Is this needed for anything? We are currently doing operations on arrays of corpora rather than using NParrallelTextCorpus directly. Therefore, are the corpus extensions needed?

No worries. I don't think any of these are things we'd actively need to use, but Damien wanted them in there more for completeness sake/for non-Serval use.
